tinc.yaml

```diff
@@ -0,0 +1,3 @@
+dependencies:
+  - name: riak-protobuf
+    path: protobuf
```

Comment:
@llelf nice, will definitely have a play with it to do a proper comparison then :) I don't think we'll be switching away from mafia any time soon as it's been built exactly for our workflow/conventions, but I'm keen to see if tinc does things any better.

Much bigger fan of this approach than stack/stackage! I'm hoping that [no-reinstall cabal](http://blog.ezyang.com/2015/09/is-no-reinstall-cabal-coming-to-ghc-8/) makes this kind of thing work even better. Not holding my breath though :)